packages/components/time-picker/src/utils.ts

```diff
@@ -66,7 +66,9 @@ export const parseDate = function (
   const day =
     isEmpty(format) || format === 'x'
       ? dayjs(date).locale(lang)
-      : dayjs(date, format).locale(lang)
+      : !format?.includes('ww')
```

Comment:
Thanks for your review. I tried to modify it again, how look right now?

Reply:
Sorry to I need to close the PR because I broke the branch 😢
so I create new PR to refresh ,Here is the Link [https://github.com/element-plus/element-plus/pull/9761](url)